api/client/trust.go

```diff
@@ -411,14 +416,17 @@ func (cli *DockerCli) trustedPush(repoInfo *registry.RepositoryInfo, tag string,
 	keys := ks.RootKeyStore().ListKeys()
 	var rootKey string
 
-	if len(keys) == 0 {
+	// TODO(dmcgowan): let user choose if multiple keys, right now just use first
+	for k := range keys {
```

Comment:
Maps in go present the order in a random way. This will not be consistent in what key it chooses.